PixiEditor/Models/Tools/Tools/MoveViewportTool.cs

```diff
@@ -18,7 +18,13 @@ public MoveViewportTool(ToolsViewModel toolsViewModel)
         }
 
         public override bool HideHighlight => true;
-        public override string Tooltip => "Move viewport. (Space)";
+        public override string Tooltip
+        {

```

Comment:
Can you make it the same way as other tools? Arrow function instead of whole `get` body?